assignments/objects.js

```diff
@@ -18,20 +19,60 @@ const example = {
 }
 
 // Write your intern objects here:
-
+const alpha = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+const bravo = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M",
+  speak: function(){
+    return "Hello, my name is Kennan!";
+  }
+}
+const charlie = {
+  "id": 3,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+}
+const delta = {
+  "id": 4,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+}
+const echo = {
+  "id": 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F",
+  multiplyNums: function(x,y){
+    return x * y;
+  }
+}
```

Comment:
It would probably make a little more sense to name these consts the name of each student. It just helps with readability with further programers.